src/themes/components/tabs.ts

```diff
@@ -1,23 +1,23 @@
-import { opacity } from '../shared/opacity';
-
 const baseStyleTab = {
   textStyle: 'body',
   transitionProperty: 'common',
   transitionDuration: 'normal',
   borderBottom: '1px',
   borderColor: 'transparent',
-  p: 'sm',
+  px: 'sm',
+  py: 'md',
```

Comment:
I am wondering, it shouldn't be 12px for px and py as the AC in the jira ticket?